lib/rpmsg/rpmsg_virtio.c

```diff
@@ -369,6 +378,11 @@ static int rpmsg_virtio_send_offchannel_nocopy(struct rpmsg_device *rdev,
 				      &rp_hdr, sizeof(rp_hdr));
 	RPMSG_ASSERT(status == sizeof(rp_hdr), "failed to write header\r\n");
 
+#ifdef VIRTIO_CACHED_BUFFERS
+	offset = metal_io_virt_to_offset(io, (void *) data);
```

Comment:
coding rule: No space is necessary after a cast  (void *)data

Reply:
ack